lib/resque/plugins/retry/hooks.rb

```diff
@@ -0,0 +1,62 @@
+module Resque
```

Comment:
@jzaleski's suggestion would mean hooks work _similar_ to how retry criteria check's work within resque-retry, I'd favour that approach.